### PR TITLE
fixes for one memory corruption, recursive include, external warning

### PR DESCRIPTION
**Pull request by andresailer**

### CODE REVIEW

File: include/process.h [hunk 1]
```diff
@@ -145,7 +145,7 @@ class JetVertexRefiner : public Algorithm {
   string _vprimcolname;
 
   //BNess tagger
-  TMVA::Reader *_bness;
+  TMVA::Reader *_bness=NULL;
```
Comment (jstrube) [inline include/process.h]: Should be nullptr ?

**Comment by jstrube** [comment]
Nice catch on the memory corruption. Thanks for submitting this.

**Comment by jstrube** [comment]
Not sure why this fails. Does this depend on Marko's travis update? I've only merged that just now.

**Comment by petricm** [comment]
The old configuration of the CI does not fork for quite some time, I have made the PR for the update of the CI nearly 6 months ago.

**Comment by jstrube** [comment]
Yes. Which I've just merged, as I said in my comment.
It looks like there's no reason why this would fail tests, so please change NULL to nullptr, and I can merge this.

**Comment by petricm** [comment]
This fails the test because it's an outdated CI, this PR is before you merged the CI update.

**Comment by andresailer** [comment]
rebased and using `nullptr`. Tests have passed.

**Comment by jstrube** [comment]
Thank you.

**Pull request merged**